berth_reservations/views.py

```diff
@@ -1,20 +1,32 @@
+import sentry_sdk
 from graphene_django.views import GraphQLView
-from sentry_sdk import capture_exception
+
+from .exceptions import VenepaikkaGraphQLError
 
 
 class SentryGraphQLView(GraphQLView):
-    def execute_graphql_request(self, *args, **kwargs):
+    def execute_graphql_request(self, request, data, query, *args, **kwargs):
         """
         Extract any exceptions and send them to Sentry
```

Comment:
Kukkuu says:
[`"""Extract any exceptions and send some of them to Sentry"""`](https://github.com/City-of-Helsinki/kukkuu/blob/develop/kukkuu/views.py#L9)
I think we should update the docsting too. Could you write something that makes more sense than their docstring? 🙈 If nothing comes to mind, just copy them.

Reply:
Whoops 😅  forgot about the docs. On it.